bc_obps/registration/api/v2/_operations/operation_id.py

```diff
@@ -0,0 +1,36 @@
+from typing import Literal, Tuple
+from uuid import UUID
+from django.http import HttpRequest
+from registration.schema.v2.operation import OperationUpdateOut, RegistrationPurposeIn
+from service.operation_service_v2 import OperationService
+from registration.constants import V2
+from registration.decorators import authorize
+
+from registration.api.utils.current_user_utils import get_current_user_guid
+
+
+from registration.decorators import handle_http_errors
+
+from registration.api.router import router
+from registration.models import (
+    Operation,
+    UserOperator,
+)
+
+from registration.schema.generic import Message
+from ninja.responses import codes_4xx
+
+
+@router.put(
+    "/v2/operations/{operation_id}/1",
```

Comment:
Or better to give the step name (operation_information) instead of the step number?

Reply:
Will change--some later steps are conditional, so numbering won't work

Reply:
@BCerki 

I agree, "better to give the step name...instead of the step number"

What do you think about this pattern:
` /v2/operations/{operation_id}/registration/purpose`
` /v2/operations/{operation_id}/registration/operation`
` /v2/operations/{operation_id}/registration/facility`
` /v2/operations/{operation_id}/registration/representative`

Reply:
Works better than step number 😉

Reply:
@marcellmueller FYI, since you are also working on endpoints for the facility information step.